src/raster/r.mblend/README.md

```diff
@@ -3,7 +3,7 @@ r.mblend
 
 Draft implementation of the [Mblend algorithm](http://www.sciencedirect.com/science/article/pii/S0098300416300012) as an add-on for [GRASS 7](https://grass.osgeo.org/grass7/). 
```

Comment:
Grass 7 here, and https://grass.osgeo.org/grass7/ redirects to https://grass.osgeo.org/learn/, so might as well lead to the homepage?
```suggestion
Draft implementation of the [Mblend algorithm](http://www.sciencedirect.com/science/article/pii/S0098300416300012) as an add-on for [GRASS](https://grass.osgeo.org). 
```

Reply:
Agreed.